napari/layers/_markers_layer/model.py

```diff
@@ -401,13 +401,17 @@ def _set_view_slice(self, indices):
         self._update()
 
     def _get_coord(self, position, indices):
```

Comment:
It's unclear to me whether clipping is the right choice here. It very well might be — but `_get_coord` should have a docstring that lets me figure that out without looking for the calling context.

Reply:
i want to fix all of the `_get_coord` stuff as mentioned in #193, and will include stuff about the clipping - but that can come up in a follow up PR